net/ocserv/files/ocserv.init

```diff
@@ -48,8 +49,7 @@ setup_config() {
 
 		ping_leases=1
 		local ifname
-		network_get_device ifname lan
-		if test -n "ifname";then
+		if network_get_device ifname lan; then
```

Comment:
@jow- suggested that in https://github.com/openwrt/packages/pull/4301#discussion_r113640821 after you had already merged that previous PR.